doc/classes/Rect2.xml

```diff
@@ -116,87 +136,100 @@
-				If [param include_borders] is [code]true[/code], they will also be considered overlapping if their borders touch, even without intersection.
+				Returns [code]true[/code] if this rectangle overlaps with the [param b] rectangle. The edges of both rectangles are excluded, unless [param include_borders] is [code]true[/code].
 			</description>
 		</method>
 		<method name="is_equal_approx" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="rect" type="Rect2" />
 			<description>
-				Returns [code]true[/code] if this [Rect2] and [param rect] are approximately equal, by calling [code]is_equal_approx[/code] on each component.
+				Returns [code]true[/code] if this rectangle and [param rect] are approximately equal, by calling [method Vector2.is_equal_approx] on the [member position] and [member size].
 			</description>
 		</method>
 		<method name="is_finite" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if this [Rect2] is finite, by calling [method @GlobalScope.is_finite] on each component.
+				Returns [code]true[/code] if this rectangle's values are finite, by calling [method Vector2.is_finite] on the [member position] and [member size].
 			</description>
 		</method>
 		<method name="merge" qualifiers="const">
 			<return type="Rect2" />
 			<param index="0" name="b" type="Rect2" />
 			<description>
-				Returns a larger [Rect2] that contains this [Rect2] and [param b].
+				Returns a [Rect2] that encloses both this rectangle and [param b] around the edges. See also [method encloses].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="end" type="Vector2" setter="" getter="" default="Vector2(0, 0)">
-			Ending corner. This is calculated as [code]position + size[/code]. Setting this value will change the size.
+			The ending point. This is usually the bottom-right corner of the rectangle, and is equivalent to [code]position + size[/code]. Setting this point also affects the [member size].
 		</member>
 		<member name="position" type="Vector2" setter="" getter="" default="Vector2(0, 0)">
-			Beginning corner. Typically has values lower than [member end].
+			The origin point. This is usually the top-left corner of the rectangle.
 		</member>
 		<member name="size" type="Vector2" setter="" getter="" default="Vector2(0, 0)">
-			Size from [member position] to [member end]. Typically, all components are positive.
-			If the size is negative, you can use [method abs] to fix it.
+			The rectangle's width and height, starting from [member position]. Setting this value also affects the [member end] point.
+			[b]Note:[/b] It's recommended setting the [member size] to a positive value, as most methods in Godot assume that the [member position] is the top-left corner, and the [member end] is the bottom-right corner. To get an equivalent rectangle with positive size, use [method abs].
```

Comment:
"setting the [member size] to a positive value": I find the wording, that a Vector is positive, a bit problematic. Do I understand it correctly, that both components of the Vector should be non-negative?

Reply:
That is correct. However, if you do find it problematic, it may in fact be. Do you have suggestions?